### ISSUE TITLE
Bug: Bloated log files in temp dir

### ISSUE THREAD

**Issue by strawlink** [JetBrains/resharper-unity#2323]
![image](https://user-images.githubusercontent.com/6575922/172352994-a0c43e65-35b1-414f-8e9d-8247d8c79dad.png)

The same log message is written millions of times, hundreds of times per second.

```
...
16:28:41.118 |I| Packages.Initialization        | :1                             | Found: 3.0.14
16:28:41.121 |I| Packages.Initialization        | :1                             | Found: 3.0.14
16:28:41.124 |I| Packages.Initialization        | :1                             | Found: 3.0.14
16:28:41.127 |I| Packages.Initialization        | :1                             | Found: 3.0.14
```
![log-spam](https://user-images.githubusercontent.com/6575922/172353637-c072687e-e2b1-4e89-9c8b-8ab99858cf82.gif)

Unity 2020.3.19f1 using plugin 3.0.14.

By looking at the modification date of the log files, I noticed they started getting bloated after I updated the package from 2.0.7 to 3.0.13.

```
JetBrains Rider 2022.1.2
Build #RD-221.5787.36, built on June 1, 2022
Subscription is active until November 23, 2022.
Runtime version: 11.0.15+10-b2043.56 amd64
VM: Dynamic Code Evolution 64-Bit Server VM by JetBrains s.r.o.
Windows 10 10.0
.NET Framework 4.0.30319.42000
GC: G1 Young Generation, G1 Old Generation
Memory: 3000M
Cores: 4
Registry:
    ide.new.project.model.index.case.sensitivity=true
    indexing.enable.entity.provider.based.indexing=false
    rdclient.asyncActions=false

Non-Bundled Plugins:
    org.intellij.RegexpTester (1.0.8)
    indent-rainbow.indent-rainbow (2.0.2)
    com.jetbrains.annotations.preloader (1.0.4)
    com.jetbrains.rider.android (221.5591.20)
    cognitivecomplexity-rider (2022.1.0)
    String Manipulation (9.4.0)
    PythonCore (221.5591.19)
    izhangzhihao.rainbow.brackets (6.22)
    zielu.gittoolbox (212.8.9)
    Jetbrains TeamCity Plugin (2022.4.108502)
    GLSL (1.20)
```

**Comment by van800** [comment]
Thank you for reporting, I have already fixed this issue in the EAP3 of Rider 2022.2.

**Comment by strawlink** [comment]
@van800 Thanks! The logging level was set to Info for me. Likely just a leftover preference from a previous bug report :)
